Day_2_Software_engineering_best_practices/solutions/07_redistributable_package/spectra_analysis/version.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, division, print_function
+from os.path import join as pjoin
+
+# Format expected by setup.py and doc/source/conf.py: string of form "X.Y.Z"
+_version_major = 0
+_version_minor = 1
+_version_micro = ''  # use '' for first of series, number for 1 and above
+_version_extra = 'dev'
+# _version_extra = ''  # Uncomment this for full releases
+
+# Construct full version string from these.
+_ver = [_version_major, _version_minor]
+if _version_micro:
+    _ver.append(_version_micro)
+if _version_extra:
+    _ver.append(_version_extra)
```

Comment:
I (in the meantime) know this is coming from shablona, but is all of the above actually needed for a minimal example of setting up a package? 
I would just do `__version__ = '0.0.1'`